aiida/orm/nodes/node.py

```diff
@@ -104,6 +104,220 @@ def iter_repo_keys(self,
                     yield key
 
 
+class NodeLinks:
```

Comment:
Perhaps this should be moved to a separate file

Reply:
Happy to do so, but I was following what you did with comments which is also in the same file. Shall I move that as well to a separate file?

Reply:
Could do, although comments was a smaller, more "trivial" bit of code, which is I why I didn't bother to use a separate file